### PR TITLE
[libpgmath] Use '/fp:fast' as floating-point behavior on Windows ARM64

### PR DESCRIPTION
**Pull request by tdusnoki**

### CODE REVIEW

File: runtime/libpgmath/lib/CMakeLists.txt [hunk 1]
```diff
@@ -149,9 +149,11 @@ elseif(${LIBPGMATH_SYSTEM_PROCESSOR} MATCHES "aarch64")
   if(${LIBPGMATH_SYSTEM_NAME} MATCHES "Windows")
     set(DEFINITIONS_L1 HOST_WIN MAXCPUS=256 MAXCPUSL=8 MAXCPUSR=8 TARGET_WIN
     TARGET_WIN_ARM64 WIN_ARM64 WINNT WIN64 TARGET_ARM64 _PGI_NOBUILTINS)
+    set(FLAGS_L1 "/fp:fast")
```
Comment (isuruf) [inline runtime/libpgmath/lib/CMakeLists.txt:152]: Use this only if `clang-cl` is used. (There's a cmake variable to see if compiler has gnu cli or not)

**Comment by kiranchandramohan** [comment]
The CI is fixed. Please rebase to run CI.

**Review by pawosm-arm — APPROVED** [review]
I assume this is an equivalent to `-ffp-contract=fast`

**Review by bryanpkc — APPROVED** [review]

**Review by isuruf — APPROVED** [review]

**Comment by kaadam** [comment]
@kiranchandramohan @bryanpkc Seems this PR is ready for landing. May I ask you to merge this PR?

**Pull request merged**